pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "assignment-07-spark"
+version = "0.1.0"
+description = "Spark homework assignment of the Data Management course at USI Università della Svizzera italiana"
+authors = ["Jeferson Morales Mariciano <jeferson.morales.mariciano@gmail.com>"]
+readme = "README.md"
+package-mode = false
+
+[tool.poetry.dependencies]
+python = "^3.10"
+pyspark = "^3.5.1"
+jupyter = "^1.0.0"
+notebook = "^7.1.2"
```

Comment:
Perhaps we should upgrade to jupyter *lab* instead? It provides a more complete interface. The pip module is `jupyterlab`

Reply:
dependencies have been updated to the richer jupyter lab environment